src/gui/tray/activitylistmodel.cpp

```diff
@@ -810,6 +811,23 @@ void ActivityListModel::slotTriggerDefaultAction(const int activityIndex)
         _currentInvalidFilenameDialog->open();
         ownCloudGui::raiseDialog(_currentInvalidFilenameDialog);
         return;
+    } else if (activity._status == SyncFileItem::FileNameClash) {
+        const auto folder = FolderMan::instance()->folder(activity._folder);
+        const auto fileName = activity._fileAction == QStringLiteral("file_renamed") ? activity._renamedFile : activity._file;
+        const auto relPath = folder ? folder->remotePath() + fileName : fileName;
```

Comment:
@claucambra Since you're returning anyways if the folder is null, you can check the `if (!folder) { return ; }` first, and then just have `const auto relPath = folder->remotePath() + fileName`

Reply:
Good point, thanks -- no longer using the conditional